docs/modules/ROOT/pages/introspection.adoc

```diff
@@ -64,5 +63,29 @@ mod Contract {
 
 Use the `supports_interface` function to query a contract's support for a given interface.
 
-TIP: If you are unsure whether the contract implements SRC5 or not, you can follow the process described in
+[,javascript]
+----
+#[starknet::contract]
+mod Contract {
+    use openzeppelin::account::interface;
+    use starknet::ContractAddress;
+
+    #[storage]
+    struct Storage {}
+
+    #[external(v0)]
+    fn target_is_account(ref self: ContractState, target: ContractAddress) {
+        let args = array![interface::ISRC6_ID];
+        let selector = selector!("supports_interface");
+
+        let mut is_account_raw = starknet::call_contract_syscall(target, selector, args.span())
```

Comment:
isn't it more idiomatic to use `ISRC5Dispatcher`?

Reply:
It certainly is :)